presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeAggregations.java

```diff
@@ -378,6 +378,21 @@ public void testReduceAgg()
         assertQuery("SELECT orderkey, array_sort(reduce_agg(linenumber, array[], (s, x) -> s || x, (s, s2) -> s || s2)) FROM lineitem GROUP BY orderkey");
     }
 
+    @Test
+    public void testNaNValueAgg()
+    {
+        // Fix Velox to get these tests passed.
+        // See https://github.com/prestodb/presto/issues/20283
+        String notEqualRowsErrorMsg = "*.not equal.*";
```

Comment:
just one nit: will you be able to paste the longer version of the expected error message in the comment?